src/Illuminate/Foundation/Console/RouteListCommand.php

```diff
@@ -506,6 +546,8 @@ protected function getOptions()
             ['sort', null, InputOption::VALUE_OPTIONAL, 'The column (domain, method, uri, name, action, middleware, definition) to sort by', 'uri'],
             ['except-vendor', null, InputOption::VALUE_NONE, 'Do not display routes defined by vendor packages'],
             ['only-vendor', null, InputOption::VALUE_NONE, 'Only display routes defined by vendor packages'],
+            ['output', 'o', InputOption::VALUE_OPTIONAL, 'Save the output to a file'],
```

Comment:
You might consider adding a little additional _optional_ information

```suggestion
            ['output', 'o', InputOption::VALUE_OPTIONAL, 'Save the output to a file (.json extension is optional)
            
            You may combine this option with --pretty for pretty printed JSON'],
```
yeah, apparently, that's how you make single line breaks in command option descriptions